ctapipe/image/image_processor.py

```diff
@@ -169,8 +171,33 @@ def _process_telescope_event(self, event):
         """
         Loop over telescopes and process the calibrated images into parameters
         """
+        location = EarthLocation.of_site("Roque de los Muchachos")
```

Comment:
Wait, are you hardcoding the ORM here for DL1Writer????

Reply:
Why do you even need a site for using `NominalFrame`?

Reply:
Wasn't meant to stay. I had trouble figuring out the position at first.